src/functionalTest/resources/scenarios/RIA-3690-remission-decision-partially-approved.json

```diff
@@ -0,0 +1,63 @@
+{
+  "description": "RIA-3690 Notification of remission decision partially approved",
+  "request": {
+    "uri": "/asylum/ccdAboutToSubmit",
+    "credentials": "AdminOfficer",
+    "input": {
+      "id": 3690,
+      "eventId": "recordRemissionDecision",
+      "state": "pendingPayment",
+      "caseData": {
+        "template": "minimal-appeal-submitted.json",
+        "replacements": {
+          "remissionDecision": "partiallyApproved",
+          "remittedAmount": "100.00",
+          "amountLeftToPay": "40.00"
+        }
+      }
+    }
+  },
+  "expectation": {
+    "status": 200,
+    "errors": [],
+    "caseData": {
+      "template": "minimal-appeal-submitted.json",
+      "replacements": {
+        "notificationsSent": [
+          {
+            "id": "3690_REMISSION_DECISION_PARTIALLY_APPROVED_ADMIN_OFFICER",
+            "value": "$/[0-9a-f]{8}-[0-9a-f]{4}-[1-5][0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}/"
+          },
+          {
+            "id": "3690_REMISSION_DECISION_PARTIALLY_APPROVED_LEGAL_REPRESENTATIVE",
+            "value": "$/[0-9a-f]{8}-[0-9a-f]{4}-[1-5][0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}/"
+          }
+        ]
+      }
+    },
+    "notifications": [
+      {
+        "reference": "3690_REMISSION_DECISION_PARTIALLY_APPROVED_LEGAL_REPRESENTATIVE",
+        "recipient": "{$TEST_LAW_FIRM_A_USERNAME}",
+        "subject": "Immigration and Asylum appeal: Remission application partially approved",
+        "body": [
+          "PA/12345/2019",
+          "CASE001",
+          "Talha Awan",
+          "{$iaExUiFrontendUrl}",
+          "40.00"
+        ]
```

Comment:
The email template in GovNotify is incorrect for Legal Representative. You need to have placeholders in the template for the customer services telephone and email address. You will check the values in the functional test here like this:

`"{$customerServices.telephoneNumber}",
  "{$customerServices.emailAddress}"`

Reply:
Added the telephoneNumber and emailAddress. Modified the template in Gov Notify